src/App.css

```diff
@@ -1,15 +1,12 @@
-.mt-3 {
+.mt-4 {
   margin-top: 2rem;
 }
 
 .mr-1 {
-  margin-right: .5rem;
+  margin-right: 0.5rem;
 }
 
-.pagination {
-  list-style: none;
-}
-
-.pagination li {
-  display: inline-block;
+.pagination-box {
```

Comment:
the word "pagination" means 

> the sequence of numbers assigned to pages in a book or periodical.

it is better to use `"some-block"` or `"some-box"`, another reason I rename to pagination-box is I added another component in it.